notifier.rst

```diff
@@ -52,18 +52,21 @@ to send SMS messages to mobile phones. This feature requires subscribing to
 a third-party service that sends SMS messages. Symfony provides integration
 with a couple popular SMS services:
 
-========  =============================  ===========================================
-Service   Package                        DSN
-========  =============================  ===========================================
-Twilio    ``symfony/twilio-notifier``    ``twilio://SID:TOKEN@default?from=FROM``
-Nexmo     ``symfony/nexmo-notifier``     ``nexmo://KEY:SECRET@default?from=FROM``
-OvhCloud  ``symfony/ovhcloud-notifier``  ``ovhcloud://KEY:SECRET@default?from=FROM``
-Sinch     ``symfony/sinch-notifier``     ``sinch://ACCOUNT_ID:AUTH_TOKEN@default?from=FROM``
-========  =============================  ===========================================
+==========  ===============================  ====================================================
+Service     Package                          DSN
+==========  ===============================  ====================================================
+Twilio      ``symfony/twilio-notifier``      ``twilio://SID:TOKEN@default?from=FROM``
+Nexmo       ``symfony/nexmo-notifier``       ``nexmo://KEY:SECRET@default?from=FROM``
+OvhCloud    ``symfony/ovhcloud-notifier``    ``ovhcloud://KEY:SECRET@default?from=FROM``
+Sinch       ``symfony/sinch-notifier``       ``sinch://ACCOUNT_ID:AUTH_TOKEN@default?from=FROM``
+FreeMobile  ``symfony/freemobile-notifier``  ``freemobile://LOGIN:PASS@default?phone=PHONE``
```

Comment:
the only line added, the rest of the diff is just indent of spaces and `=`

Reply:
The bottom separator has one `=` too much in the first column:

```rst
==========  ===============================  ====================================================
Service     Package                          DSN
==========  ===============================  ====================================================
...
===========  ==============================  ====================================================
```